SPECS/qt-rpm-macros/qt-rpm-macros.spec

```diff
@@ -1,48 +1,51 @@
 Summary:        RPM macros for building Qt5 and KDE Frameworks 5 packages
-Name:           qt5-rpm-macros
-Version:        5.12.5
-Release:        3%{?dist}
+Name:           qt-rpm-macros
```

Comment:
Comment doesn't really apply to this file, but are the nopatch files still relevant?  We can probably remove them

Reply:
I went through and checked each .nopatch file present for qtsvg. We have upgrade past all of the affected versions according to the NIST, so these can are removed as noted.